manim/mobject/svg/svg_mobject.py

```diff
@@ -257,36 +286,44 @@ def attribute_to_float(self, attr):
         )
         return float(stripped_attr)
 
-    def polygon_to_mobject(self, polygon_element):
+    def polygon_to_mobject(self, polygon_element, style):
```

Comment:
```suggestion
    def polygon_to_mobject(self, polygon_element: MinidomElement, style: dict):
```

Reply:
Should the type be the imported name, or some full path?
I may have to update the other references from `minidom.Element` to MinidomElement if this change is correct.

Reply:
the function header should be the same as how you imported.
but in the docstring, it needed to be so, it should be `minidom.Element`